src/rpcblockchain.cpp

```diff
@@ -798,6 +798,8 @@ UniValue getmempoolinfo(const UniValue& params, bool fHelp)
             "  \"size\": xxxxx                (numeric) Current tx count\n"
             "  \"bytes\": xxxxx               (numeric) Sum of all tx sizes\n"
             "  \"usage\": xxxxx               (numeric) Total memory usage for the mempool\n"
+            "  \"maxmempool\": xxxxx          (numeric) Maximum memory usage for the mempool\n"
```

Comment:
Nit: all the other RPC helps have commas after all but the last field in an object, to make it look more like actual JSON:
```
"  \"usage\": xxxxx,
"  \"maxmempool\": xxxxx,
"  \"mempoolminfee\": xxxxx
```
Not that I care deeply about this (or at all, really), but you may preempt someone that wants to make that update later :)